### PR TITLE
feat: photovoltaic experiment, series and parallel connection

### PR DESCRIPTION
**Pull request by saidabdul80**

### CODE REVIEW

File: routes/web.php [hunk 1]
```diff
@@ -25,7 +25,8 @@
 Route::get('/create-course', 'App\Http\Controllers\CreateCourse@index')->name('create-courses');
 
 
-
+ 
+Route::get('/vicelab-exp2','App\Http\Controllers\experimentController@vicelabexp2')->name('vicelab-exp2');
```
Comment (Auwalms) [inline routes/web.php]: @saidabdul80 please use a more descriptive url path. not sure `/vicelab-exp2` looks good like the others.

**Review by Auwalms — CHANGES_REQUESTED** [review]

File: routes/web.php (hunk 1 above)
Comment (saidabdul80) [reply routes/web.php]: Ok

**Review by Auwalms — APPROVED** [review]
Looks good

File: app/Http/Controllers/experimentController.php [hunk 2]
```diff
@@ -18,4 +18,12 @@ public function simplePendulum()
     {
     	return view('experiment/simple_pendulum');
     }
+    public function vicelabexp2()
```
Comment (abdullahikawu) [inline app/Http/Controllers/experimentController.php:21]: ViceLab? Again use a better name for this method

File: app/Http/Controllers/experimentController.php [hunk 3]
```diff
@@ -18,4 +18,12 @@ public function simplePendulum()
...
+    {
+        return view('experiment/vicelabexp2');
+    }
+    public function vicelabexp3()
```
Comment (abdullahikawu) [inline app/Http/Controllers/experimentController.php:25]: Use a descriptive name for function

File: routes/web.php [hunk 4]
```diff
@@ -34,7 +34,9 @@
 
 
 
-
+ 
+Route::get('/photovoltaic-2','App\Http\Controllers\experimentController@vicelabexp2')->name('vicelab-exp2');
```
Comment (abdullahikawu) [inline routes/web.php:38]: Also check the part URL

Comment (bynalab) [reply routes/web.php:38]: Like we discussed, @saidabdul80, you can use the new way of routing 
Like:
Route::get('/photovoltaic', [experimentController::class, 'vicelabexp2']);

**Comment by Auwalms** [comment]
Closing this since https://github.com/cs-dept-ibbul/vlab/pull/56 has it covered with all the comments mentioned here effected

**Pull request closed without merging**